src/app/components/record-content/record-content.component.scss

```
@@ -0,0 +1,48 @@
+.rContent{
+    border: 3px solid grey;
+    width: 800px;
+    height: 450px;
+    border-radius: 10px;
+    overflow-y: scroll;
+    margin: auto;
+    li{
+        list-style: none;
+    }
+
+    ul{
+        padding-top: 20px;
+        padding-bottom: 20px;
+    }
+}
+
+
+.rContent::-webkit-scrollbar {
+    width: 7px;              /* width of the entire scrollbar */
```

```suggestion
    width: 7px;
```
Do not leave unnecessary comments